opentreemap/importer/views.py

```diff
@@ -110,6 +113,18 @@ def get_import_table(request, instance, table_name):
 _EVENT_TABLE_PAGE_SIZE = 5
 
 
+def _cleanup_tables(instance):
+    ievents = (list(TreeImportEvent.objects.filter(instance=instance)) +
+               list(SpeciesImportEvent.objects.filter(instance=instance)))
+    for ie in ievents:
+        if ie.task_id != '':
```

Comment:
Since we previously did not unset `task_id` after an import finished, should we exclude imports that are in the `FINISHED_CREATING` state, so that we do not move finished imports into an `UNKNOWN_ERROR` incorrectly?